apptest/tests/vmagent_remotewrite_test.go

```diff
@@ -63,6 +88,9 @@ func TestSingleVMAgentReloadConfigs(t *testing.T) {
 
 	vmagent.ReloadRelabelConfigs(t)
 
+	checkResponse(vmagentAddr+"/remotewrite-url-relabel-config", "replacement: value2")
```

Comment:
The purpose of `TestSingleVMAgentReloadConfigs` differs from testing API calls. Maybe a separate test dedicated to configuration validation should be added. I’m not entirely sure, since it adds extra cost, but the alternative, broadening the current test scope, isn’t ideal either.